Dangerfile

```diff
@@ -1,11 +1,8 @@
-# Make it more obvious that a PR is a work in progress and shouldn't be merged yet
-warn("PR is classified as Work in Progress. When complete, please remove the [WIP] and assign reviewers.") if github.pr_title.include? "[WIP]"
-
-warn "Please link to a GitHub issue within this pull request. Example: Closes #9999" unless (github.pr_body.include? "Closes #" or github.pr_body.include? "References #")
+warn "Please link to a GitHub issue within this pull request. Example: Closes #9999" unless (github.pr_body.downcase.include? "closes #" or github.pr_body.downcase.include? "references #")
 
 # Add in our standard using GitHub Closes within the Pull request body.
-warn "Please add a Changes section to your pull request description." unless github.pr_body.include? "Changes"
-warn "Please add a Purpose section to your pull request description." unless github.pr_body.include? "Purpose"
+warn "Please add a Changes section to your Pull Request description." unless github.pr_body.downcase.include? "changes"
```

Comment:
I'm glad there's an actual `.downcase`!

Reply:
Yep just some ruby. Small scripts like this are perfect for the lang.